act-framework.bs

```diff
@@ -63,7 +63,7 @@ ACT Test Procedure {#test-proc}
 Selector {#test-proc-selector}
 --------
 
-A selector is a boolean predicate that takes an element or component in the set represented in the input data and unambiguously tests whether the element or component matches the selector or not. In an ACT Rule, a selector is applied to the entire input data to filter it into a set of elements or components to be evaluated with the test procedure.
+A selector is a pattern used to select element(s) in an HTML document or node(s) in the DOM tree. In an ACT Rule, a selector is applied to the entire input data to filter it into a set of elements or nodes to be evaluated against the test procedure.
```

Comment:
I prefer @WilcoFiers' wording in that it's more general, but with still a couple nitpicks:

- I don't think a selector is a `procedure` (i.e. a series of actions), it's more a _declarative thing_ used in a procedure.
- strictly speaking, the selector isn't used to _locate_ elements, but in our case to _include_ them in the test set, or in other words to _filter_ the input data. The "location" information is secondary and not always relevant.